build.zig

```diff
@@ -5,6 +5,8 @@ const TracerBackend = enum {
     none,
     // Writes to a file (./tracer.json) which can be uploaded to https://ui.perfetto.dev/
     perfetto,
+    // Sends data to https://github.com/wolfpld/tracy.
+    tracy,
```

Comment:
The "Which backend to use for tracing" needs to be updated to include tracy. Or the "One of: ..." part can just be removed; I think `zig build -h` automatically lists the possible options for enums.